### ISSUE TITLE
Nested checkboxes and radios don't get selected by default value anymore

### ISSUE THREAD

**Issue by Tim-Wils** [verbb/formie#1661]
### Describe the bug

Having a fieldgroup with nested checkbox groups or radio groups with default values don't get checked anymore.
I found out the change in formie templates for groups in this commit is the problem: https://github.com/verbb/formie/commit/cad299b36a225ad778d9fdca7d8726c9573afec7 
There's no value passed along anymore, so it's always emty in radio.html and checkbox.html.

If I change the create on this line:
```
element: block ?? create({
    class: 'verbb\\formie\\elements\\NestedFieldRow',
    fieldId: groupField.id,
}),
```
back to null it works:
`element: block ?? null,`

I tried passing the value separately like so, but it looked like it only worked for checkboxgroups:
```
{{ craft.formie.renderField(form, field, {
      value: value,
      element: block ?? create({
          class: 'verbb\\formie\\elements\\NestedFieldRow',
          fieldId: groupField.id,
      }),
  }) }}
```

### Steps to reproduce

1. Create a nested group with multiple radio's / checkboxes that have a value is default.
2. Do the same for non-nested radio/checks.
3. Watch the front-end and the nested defaults won't be selected whereas the non-nested will be.

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.5.12

### Plugin version

2.0.44.1

### Multi-site?

no

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.44.1"`.

**Comment by engram-design** [comment]
Fixed in [2.0.45](https://github.com/verbb/formie/releases/tag/2.0.45)
